### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lucca65**
## What issue does this PR close
Closes N/A

## Changes Proposed ( a list of new changes introduced by this PR)
- Update docs for configuring our Nginx redirects, to include new Mailer configuration together with existing API 

## How to test ( a list of instructions on how to test this PR)
- Configure a new nginx server using the current configs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,6 +167,39 @@ Note: Replace `#{backend_listening_port}` with the port to which the backend ser
 
 This code redirects the identified crawler to the correct rich link URL. Nut if a crawler was not detected the server runs normally.
 
+## Nginx redirect setup
```
Comment (MatheusBuss) [inline README.md:170]: To make it consistent
```suggestion
## NGINX redirect setup
```

File: README.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
 #Boom! Now you can hack away!
 
-## NGINX setup
+## NGINX crawlers setup
 
 We use NGINX to manage our server infrastructure. You can check out how to install and run it on their [official site](https://nginx.org/)
```
Comment (MatheusBuss) [inline README.md:131]: Perhaps we should have a section about NGINX and inside this section a topic about the crawlers and another for the redirects
```suggestion
## NGINX

We use NGINX to manage our server infrastructure. You can check out how to install and run it on their [official site](https://nginx.org/)

## NGINX crawlers setup
```

**Review by MatheusBuss — COMMENTED** [review]
I think it makes more sense to invert the order in which the NGINX sections appear. First the general redirects and then the crawlers redirects. But I think that either way is fine.

**Review by MatheusBuss — APPROVED** [review]

**Review by MatheusBuss — APPROVED** [review]
:+1:

**Comment by lucca65** [comment]
@MatheusBuss, I've followed your suggestion, looks much better. I've also added section links

**Comment by NeoVier** [comment]
> Add emoji for good measure

🤣

File: README.md [hunk 3]
```diff
@@ -124,13 +124,47 @@ mix phx.server
...
+                proxy_http_version 1.1;
+                proxy_set_header Connection "upgrade";
+                proxy_set_header Origin *;
+                proxy_pass http://ADDRESS:PORT;
+                client_max_body_size CLIENT_SIZE;
+        }
+```
+## NGINX Setup: Crawlers
+
+Cambiatus has builtin support for social media crawlers, in order to dynamically serve rich links (or Open Graphs) in accordance to the [Open Graph Protocol](https://ogp.me/). To setup, follow this steps:
```
Comment (MatheusBuss) [inline README.md:167]:
```suggestion
Cambiatus has builtin support for social media crawlers, in order to dynamically serve rich links (or Open Graphs) in accordance to the [Open Graph Protocol](https://ogp.me/). To setup, follow these steps:
```

Comment (MatheusBuss) [reply README.md:167]: Looks great to me! Just a grammatical correction.

**Review by MatheusBuss — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (lucca65) [reply README.md:167]: ops! Thanks!

**Pull request merged**